ui/components/app/multilayer-fee-message/multi-layer-fee-message.js

```diff
@@ -91,15 +95,15 @@ export default function MultilayerFeeMessage({
         key="total-item-gas-fee"
         detailTitle={t('layer1Fees')}
         detailTotal={layer1Total}
-        detailText={feeTotalInFiat}
+        detailText={useCurrencyRateCheck && feeTotalInFiat}
         noBold={plainStyle}
         flexWidthValues={plainStyle}
       />
       <TransactionDetailItem
         key="total-item-total"
         detailTitle={t('total')}
         detailTotal={totalInEth}
-        detailText={totalInFiat}
+        detailText={useCurrencyRateCheck && totalInFiat}
```

Comment:
maybe we can move or add the useCurrencyRateCheck checks to where we define the values so that when we declare the variables, we won't need to construct the new JSX elements if they are not needed

Reply:
done